js/Models/Notifications/ExerciseEndNotificationScenario.ts

```diff
@@ -0,0 +1,25 @@
+/// <reference path="./INotificationScenario.d.ts" />
+/// <reference path="./INotification.d.ts" />
+/// <reference path="../IWorkoutPlanAction.d.ts" />
+/// <reference path="../IExcercisePlanAction.d.ts"/>
```

Comment:
No need to add references to *.d.ts* files as *tsconfig.json* already lists all those files